From 61f91f7c22721d8d65b5f760fc6b091e1a0289c4 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 29 Jul 2020 08:05:49 +0300
Subject: [PATCH 030/113] Revert "FODCircleView: Let us call show() when
 mIsBouncer() && !isPinOrPattern()"

This reverts commit 5fa02eab60af8fccded11911391eedb3f3d20299.
---
 .../src/com/android/systemui/biometrics/FODCircleView.java    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
index 6be2fef143d..38f89549f0c 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
@@ -324,8 +324,8 @@ public class FODCircleView extends ImageView {
             return;
         }
 
-        if (mIsBouncer && !isPinOrPattern(mUpdateMonitor.getCurrentUser())) {
-            // Ignore show calls when Keyguard password screen is being shown
+        if (mIsBouncer) {
+            // Ignore show calls when Keyguard pin screen is being shown
             return;
         }
 
-- 
2.25.1

